src/cron/schedule-code-review.js

```diff
@@ -88,6 +90,11 @@ const checkThreadsForReview = async (client, statusText) => {
     }
   } catch (error) {
     saveErrorLog(error);
+    sendErrorToChannel(
+      client,
+      translateLanguage('sendChannelError.error'),
```

Comment:
I noticed that always that you use this function you pass this same translation, maybe you can hardcode it inside of the function so you don't have to pass it for params each time

Reply:
What about in the function have that  translation but we pass in the error param context information or more info to clarify the users

something like "you can pass this location to the technical service:" and pass the name of the function or more clear error message